packages/cli/src/commands/bisect/index.ts

```diff
@@ -56,6 +54,9 @@ export default async function bisect(client: Client): Promise<number> {
   telemetry.trackCliOptionRun(parsedArgs.flags['--run']);
   telemetry.trackCliFlagOpen(parsedArgs.flags['--open']);
 
+  const scope = await getScope(client);
```

Comment:
Moved scope lower in the order of execution. If it remains at the top we check for someone to be authetincated before allowing them to run `vercel bisect --help` which feels like a bug.

Reply:
Extracted to https://github.com/vercel/vercel/pull/12439